From bc234605fa3eb10f56bf0d74c8ecb0d91ada9d05 Mon Sep 17 00:00:00 2001
From: Damien Miller <djm@mindrot.org>
Date: Wed, 7 Dec 2022 18:38:25 +1100
Subject: [PATCH] disable SANDBOX_SECCOMP_FILTER_DEBUG

It was mistakenly enabled in 2580916e4872

Reported by Peter sec-openssh-com.22.fichtner AT 0sg.net
---
 sandbox-seccomp-filter.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sandbox-seccomp-filter.c b/sandbox-seccomp-filter.c
index 9929bdb6851..cec43c463ea 100644
--- a/sandbox-seccomp-filter.c
+++ b/sandbox-seccomp-filter.c
@@ -23,7 +23,7 @@
  * E.g.
  *   auditctl -a task,always -F uid=<privsep uid>
  */
-#define SANDBOX_SECCOMP_FILTER_DEBUG 1
+/* #define SANDBOX_SECCOMP_FILTER_DEBUG 1 */
 
 #if 0
 /*
